spec/dap/utils/misc_spec.rb

```diff
@@ -0,0 +1,10 @@
+describe Dap::Utils::Misc do
+  context 'flatten_hash' do
```

Comment:
In the future this should be `describe '.flatten_hash'` ... `context` blocks generally start with `with` like 
```
context "with a stubbed class" do
end
```